app/actors/sufia/default_admin_set_actor.rb

```diff
@@ -25,7 +25,7 @@ def ensure_admin_set_attribute!(attributes)
       end
 
       def ensure_permission_template!(admin_set_id:)
-        Sufia::PermissionTemplate.find_or_create_by!(admin_set_id: admin_set_id)
+        Sufia::PermissionTemplate.find_by(admin_set_id: admin_set_id) || create_permission_template!(admin_set_id: admin_set_id)
```

Comment:
In Hyrax, there is a somewhat tenuous assumption that PermissionTemplate and AdminSet combine to a larger concept; The PermissionTemplate is the ephemeral information and "calculations" that are applied to items added to an AdminSet. As such, I believe every AdminSet should have one and only one PermissionTemplate (and enforce its creation if one is absent).